tests/test_core/test_alignment.py

```diff
@@ -985,6 +981,56 @@ def test_construction(self):
         seq2 = make_seq("AC", name="seq2")
         coll = SequenceCollection(data=[seq1, seq2])
 
+    def test_repr(self):
```

Comment:
We are gradually converting existing tests to pytest style. Can you cut this method at the bottom of the file and edit as follows
```python
@pytest.mark.parametrize("cls", [SequenceCollection, Alignment, ArrayAlignment])
def test_repr(cls):
    ...
    seqs = cls(data=data, moltype=DNA)
    # replace all self.assertEqual calls as per
    assert repr(seqs) == "2x (ENSMUSG00000039616[GCCCTTCAAAT...], ENSMUSG00000056468[GCCAGGGGGGA...]) <dna> seqcollection"
```
etc...

Reply:
This was written in only the `SequenceCollectionTests` class so I've resolved this without parametrizing, output for `Alignment` and `ArrayAlignment` should be different